cmd/management.go

```diff
@@ -136,17 +137,18 @@ Output:
 }
 
 var listApplicationsCmd = &cobra.Command{
-	Use:   "applications [name]",
+	Use:   "applications [project] [name]",
 	Short: "Lists applications",
-	Long:  "Lists available applications. Optional parameter allows to return only the application with the given name.",
+	Long: `Lists available applications for the project. 
+		Optional parameter [name] allows to return only the application with the given name.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		withLogin(cmd.Context(), func(ctx context.Context) error {
-			resp, err := client.ManagementGet().ListApplications(ctx)
+			resp, err := client.ManagementGet().ListApplications(ctx, args[0])
```

Comment:
You need to check that if there is at least one parameter passed if it's optional,
or change [project] to {project} and set `Args:  cobra.MinimumNArgs(1),`

Reply:
thanks fixed.